src/services/impl/FileTextReader.hh

```diff
@@ -0,0 +1,52 @@
+#ifndef RA_SERVICES_FILETEXTREADER
+#define RA_SERVICES_FILETEXTREADER
+#pragma once
+
+#include "services\TextReader.hh"
+
+#include "RA_StringUtils.h"
+
+#include <fstream>
+
+namespace ra {
+namespace services {
+namespace impl {
+
+class FileTextReader : public ra::services::TextReader
+{
+public:
+    FileTextReader(const std::wstring& sFilename)
```

Comment:
Default constructors need to marked with `noexcept`. Default constructors with parameters need to be marked with explicit to prevent `implicit `conversion.

Reply:
ok